english/chopper/get-to-it.md

```diff
@@ -0,0 +1,3 @@
+The bananas were hot while the tea was cold. What craziness is this? Give me a current bun and be done with it all, I thought. At least I don't have to get to the [chopper](http://s2.quickmeme.com/img/98/982c902d16cad216960055a63f97c0f776ef6745d52c9713a2c5e8c5a07ce115.jpg)
+
+Alternatively I could always go [here](https://github.com/leogovan/create-your-own-adventure/blob/master/english/be-octodog/octodog.md).  
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.